deploy/src/main/deploy/example-credentials.json

```diff
@@ -54,6 +54,17 @@
             "key" : "Y29hcC1zZWNyZXQ="
           }
         ]
+      },
+      {
+        "device-id": "4711",
+        "type": "x509-cert",
+        "auth-id": "CN=Device 4711,OU=Hono,O=Eclipse IoT,L=Ottawa,C=CA",
```

Comment:
Since we already have the mechanism in place, can't we also use it to inject the device cert's subject DN to these credentials? IMHO that would make the whole process less error prone when changing or adding a device ...

Reply:
Of course. That's good.